lib/source-map/util.js

```diff
@@ -185,6 +185,10 @@ define(function (require, exports, module) {
   }
   exports.join = join;
 
+  exports.isAbsolute = function (aPath) {
```

Comment:
Should [`normalize`][1] use this new utility instead of having it own similar check?

[1]: https://github.com/fitzgen/source-map/blob/source-content-bug/lib/source-map/util.js#L93